<ui version="4.0" >
 <class>XUPProjectManager</class>
 <widget class="QWidget" name="XUPProjectManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Project Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="spacing" >
       <number>2</number>
      </property>
      <property name="margin" >
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbOpen" >
        <property name="iconSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbClose" >
        <property name="iconSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>259</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="tbPages" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="wCurrentProject" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>330</width>
        <height>145</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Current Project</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QTreeView" name="tvCurrentProject" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wOpenedProjects" >
      <attribute name="label" >
       <string>Opened Projects</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QListWidget" name="lwOpenedProjects" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wProjectsNative" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>319</width>
        <height>145</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Native Projects</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QTreeView" name="tvNative" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pteLog" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
